### PR TITLE
Bump version to v0.2.0

### PR DESCRIPTION
**Pull request by HAOCHENYE**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +56,16 @@ Major features:
...
 
+## What's New
+
+v0.2.0 was released in 2022-10-10.
+
+1. Refactor FileIO, more unified interface, no BC breaking.
+2. Add base model for test time augmentation.
+3. Support convert `BN` to `SyncBN` during distributed data parallel training.
+
+Read[Changelog](./docs/en/notes/changelog.md#v0.2.0 "10/10/2022") for more details.
```
Comment (zhouzaida) [inline README.md:67]:
```suggestion
Read [Changelog](./docs/en/notes/changelog.md#v0.2.0 "10/10/2022") for more details.
```

File: README_zh-CN.md [hunk 2]
```diff
@@ -58,6 +58,16 @@ MMEngine 是一个基于 PyTorch 用于深度学习模型训练的基础库，
    - 对神经网络模型中各个层的优化配置进行细粒度调整
    - 混合精度训练的灵活控制
 
+## 最近进展
+
+最新版本 v0.2.0 在 2022.10.10 发布。
+
+1. FileIO 重构，提供更加统一的接口，且向下兼容
```
Comment (zhouzaida) [inline README_zh-CN.md:65]:
```suggestion
1. 重构 FileIO 以提供更加易用的接口并保持向下兼容
```

File: README.md [hunk 3]
```diff
@@ -56,6 +56,16 @@ Major features:
...
 
+## What's New
+
+v0.2.0 was released in 2022-10-11.
+
+1. Refactor FileIO, more unified interface, no BC breaking.
+2. Add the base model for test time augmentation.
+3. Support converting `BN` to `SyncBN` during distributed data-parallel training.
+
+Read [Changelog](./docs/en/notes/changelog.md#v0.2.0 "10/10/2022") for more details.
```
Comment (zhouzaida) [inline README.md:67]:
```suggestion
Read [Changelog](./docs/en/notes/changelog.md#v0.2.0 "11/10/2022") for more details.
```

File: docs/en/notes/changelog.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+- @okotaku made their first contribution in https://github.com/open-mmlab/mmengine/pull/510
+- @MengzhangLI made their first contribution in https://github.com/open-mmlab/mmengine/pull/527
+- @MambaWong made their first contribution in https://github.com/open-mmlab/mmengine/pull/536
+- @cxiang26 made their first contribution in https://github.com/open-mmlab/mmengine/pull/548
+- @nijkah made their first contribution in https://github.com/open-mmlab/mmengine/pull/549
+- @Zhengfei-0311 made their first contribution in https://github.com/open-mmlab/mmengine/pull/562
+- @austinmw made their first contribution in https://github.com/open-mmlab/mmengine/pull/579
+- @yhna940 made their first contribution in https://github.com/open-mmlab/mmengine/pull/569
+- @liuyanyi made their first contribution in https://github.com/open-mmlab/mmengine/pull/539
```
Comment (zhouzaida) [inline docs/en/notes/changelog.md:79]: This section can be removed.

File: docs/en/notes/changelog.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+- Fix detect_anomalous_params by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/588
+
+- Some out-of-date unit tests by @C1rN09 in https://github.com/open-mmlab/mmengine/pull/586
+
+- Add smddp dist backend option by @austinmw in https://github.com/open-mmlab/mmengine/pull/579
+
+- Fix typo by @yhna940 in https://github.com/open-mmlab/mmengine/pull/569
+
+- Fix loss smooth when the loss name doesn't start with `loss` by @liuyanyi in
+  https://github.com/open-mmlab/mmengine/pull/539
```
Comment (zhouzaida) [inline docs/en/notes/changelog.md:67]: All blank lines can be removed.

**Review by zhouzaida — APPROVED** [review]

File: docs/en/notes/changelog.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+- Remove unnecessary "if statement" by @MambaWong in https://github.com/open-mmlab/mmengine/pull/536
+- Fix CheckpointHook behavior unexpected if given `filename_tmpl` argument by @C1rN09 in https://github.com/open-mmlab/mmengine/pull/518
+- Fix `_save_to_state_dict` by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/542
+- Support compare NumPy array dataset meta by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/511
+- Use `get` instead of `pop` to dump `runner_type` by @nijkah in https://github.com/open-mmlab/mmengine/pull/549
+- Upgrade pre-commit hooks by @zhouzaida in https://github.com/open-mmlab/mmengine/pull/576
+- Delete the error comment by @vansin in https://github.com/open-mmlab/mmengine/pull/514
+- Fix detect_anomalous_params by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/588
+- Some out-of-date unit tests by @C1rN09 in https://github.com/open-mmlab/mmengine/pull/586
+- Add smddp dist backend option by @austinmw in https://github.com/open-mmlab/mmengine/pull/579
```
Comment (ZwwWayne) [inline docs/en/notes/changelog.md:42]: SMDDP，it is a feature and might need to be highlighted, because it makes it can run on AWS

File: docs/en/notes/changelog.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+- Fix unit test in windows by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/515
+- Fix merge ci & multiprocessing unit test by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/529
+- Fix unit test could fail caused by `MultiProcessTestCase`  by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/535
+- Remove unnecessary "if statement" by @MambaWong in https://github.com/open-mmlab/mmengine/pull/536
+- Fix CheckpointHook behavior unexpected if given `filename_tmpl` argument by @C1rN09 in https://github.com/open-mmlab/mmengine/pull/518
+- Fix `_save_to_state_dict` by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/542
+- Support compare NumPy array dataset meta by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/511
+- Use `get` instead of `pop` to dump `runner_type` by @nijkah in https://github.com/open-mmlab/mmengine/pull/549
+- Upgrade pre-commit hooks by @zhouzaida in https://github.com/open-mmlab/mmengine/pull/576
+- Delete the error comment by @vansin in https://github.com/open-mmlab/mmengine/pull/514
```
Comment (ZwwWayne) [inline docs/en/notes/changelog.md:39]: small issues can be skipped.

File: docs/en/notes/changelog.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+- Fix detect_anomalous_params by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/588
+- Some out-of-date unit tests by @C1rN09 in https://github.com/open-mmlab/mmengine/pull/586
+- Add smddp dist backend option by @austinmw in https://github.com/open-mmlab/mmengine/pull/579
+- Fix typo by @yhna940 in https://github.com/open-mmlab/mmengine/pull/569
+- Fix loss smooth when the loss name doesn't start with `loss` by @liuyanyi in
```
Comment (ZwwWayne) [inline docs/en/notes/changelog.md:44]: what does fix loss smooth mean?

File: docs/en/notes/changelog.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+- Support defining metric name in wandb backend @okotaku in https://github.com/open-mmlab/mmengine/pull/509
+- Use `torch.lerp\_()` to speed up EMA by @RangiLyu in https://github.com/open-mmlab/mmengine/pull/519
+- Add test time augmentation base model by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/538
+- Support converting `BN` to `SyncBN` by config by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/506
+- Refactor `FileIO` but without breaking bc by @zhouzaida in https://github.com/open-mmlab/mmengine/pull/533
+- Add dockerfile by @zhouzaida in https://github.com/open-mmlab/mmengine/pull/347
+
+### Docs
+
+- Fix API files of en docs by @zhouzaida in https://github.com/open-mmlab/mmengine/pull/525
```
Comment (ZwwWayne) [inline docs/en/notes/changelog.md:16]: en docs -> English documentation
